core/string/src/common.rs

```diff
@@ -214,735 +221,730 @@ const MAX_STATIC_LENGTH: usize = {
-    JsStr::latin1("dateAdd".as_bytes()),
-    JsStr::latin1("dateUntil".as_bytes()),
-    JsStr::latin1("era".as_bytes()),
-    JsStr::latin1("eraYear".as_bytes()),
-    JsStr::latin1("fields".as_bytes()),
-    JsStr::latin1("mergeFields".as_bytes()),
+    StaticString::new(JsStr::latin1("get Id".as_bytes())),
+    StaticString::new(JsStr::latin1("getOffsetNanosecondsFor".as_bytes())),
+    StaticString::new(JsStr::latin1("getOffsetStringFor".as_bytes())),
+    StaticString::new(JsStr::latin1("getPlainDateTimeFor".as_bytes())),
+    StaticString::new(JsStr::latin1("getInstantFor".as_bytes())),
+    StaticString::new(JsStr::latin1("getPossibleInstantFor".as_bytes())),
+    StaticString::new(JsStr::latin1("getNextTransition".as_bytes())),
+    StaticString::new(JsStr::latin1("getPreviousTransition".as_bytes())),
+    StaticString::new(JsStr::latin1("id".as_bytes())),
+    StaticString::new(JsStr::latin1("Now".as_bytes())),
+    StaticString::new(JsStr::latin1("Calendar".as_bytes())),
+    StaticString::new(JsStr::latin1("Duration".as_bytes())),
+    StaticString::new(JsStr::latin1("Instant".as_bytes())),
+    StaticString::new(JsStr::latin1("PlainDate".as_bytes())),
+    StaticString::new(JsStr::latin1("PlainDateTime".as_bytes())),
+    StaticString::new(JsStr::latin1("PlainMonthDay".as_bytes())),
+    StaticString::new(JsStr::latin1("PlainTime".as_bytes())),
+    StaticString::new(JsStr::latin1("PlainYearMonth".as_bytes())),
+    StaticString::new(JsStr::latin1("TimeZone".as_bytes())),
+    StaticString::new(JsStr::latin1("ZonedDateTime".as_bytes())),
+    StaticString::new(JsStr::latin1("timeZoneId".as_bytes())),
+    StaticString::new(JsStr::latin1("instant".as_bytes())),
+    StaticString::new(JsStr::latin1("plainDateTime".as_bytes())),
+    StaticString::new(JsStr::latin1("plainDateTimeISO".as_bytes())),
+    StaticString::new(JsStr::latin1("zonedDateTime".as_bytes())),
+    StaticString::new(JsStr::latin1("zonedDateTimeISO".as_bytes())),
+    StaticString::new(JsStr::latin1("plainDate".as_bytes())),
+    StaticString::new(JsStr::latin1("plainDateISO".as_bytes())),
+    StaticString::new(JsStr::latin1("get epochSeconds".as_bytes())),
+    StaticString::new(JsStr::latin1("get epochMilliseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("get epochMicroseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("get epochNanoseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("epochSeconds".as_bytes())),
+    StaticString::new(JsStr::latin1("epochMilliseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("epochMicroseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("epochNanoseconds".as_bytes())),
+    StaticString::new(JsStr::latin1("subtract".as_bytes())),
+    StaticString::new(JsStr::latin1("until".as_bytes())),
+    StaticString::new(JsStr::latin1("since".as_bytes())),
+    StaticString::new(JsStr::latin1("equals".as_bytes())),
+    StaticString::new(JsStr::latin1("toZonedDateTime".as_bytes())),
```

Comment:
thought: temporal will probably always be `temporal` feature flagged because of the dependency, so should we also be feature flagging the `StaticString`s that we register.

Reply:
Actually that's very true.